pkg/types/interfaces.go

```diff
@@ -27,6 +27,7 @@ type KafkaSourceClient interface {
 	CreateKafkaSource(kafkaSource *v1alpha1.KafkaSource) error
 	DeleteKafkaSource(name string) error
 	GetKafkaSource(name string) (*v1alpha1.KafkaSource, error)
+	GetKafkaSources() (*v1alpha1.KafkaSourceList, error)
```

Comment:
For the client API we used `ListKafkaSources` and I think it would make sense to you the same naming conventions here, also to avoid confusion with the singular versions of this method.